### ISSUE TITLE
iquery should not interpret GenQuery column names

### ISSUE THREAD

**Issue by trel** [irods/irods_client_library_rirods#17]
All of the columns available to be returned by a GenQuery are possible inputs to `iquery`.

It should return the column names as is, without interpretation... as there are many...

```
$ iquest attrs | wc -l
327
```

**Comment by MartinSchobben** [comment]
Fixed by PR #19. It now produces a matrix without column names. Would this be desired?

From the README:
```
> iquery("SELECT COLL_NAME, DATA_NAME WHERE DATA_NAME LIKE 'foo%'")
     [,1]                   [,2]     
[1,] "/tempZone/home/bobby" "foo.csv"
[2,] "/tempZone/home/bobby" "foo.rds"
```

**Comment by trel** [comment]
I think so - should be nice and generic.  Closing.
